a10_octavia/controller/worker/tasks/vthunder_tasks.py

```diff
@@ -454,6 +530,9 @@ def execute(self, vthunder):
                                                      interval, timeout, max_retries, method, url,
                                                      expect_code, port, ipv4)
             LOG.debug("Successfully created health monitor for vThunder %s", vthunder.id)
+        except acos_errors.Exists:
+            LOG.debug("health monitor slb server already exist on vThunder %s", vthunder.id)
+            pass
```

Comment:
If we have a log then we don't need a pass.

Reply:
done